tests/e2e/tidbcluster/stability.go

```diff
@@ -1591,5 +1591,27 @@ var _ = ginkgo.Describe("[Stability]", func() {
 			err = cli.PingcapV1alpha1().TidbClusterAutoScalers(tcas.Namespace).Delete(tcas.Name, &metav1.DeleteOptions{})
 			framework.ExpectNoError(err, "failed to delete auto-scaler")
 		})
+
+		// TODO: remove helm deployed backup/restore, use CR instead
```

Comment:
We can just remove this case and there is case for BR with CR in `stability-br.go`.

Reply:
👌